selfdrive/car/volkswagen/values.py

```diff
@@ -123,7 +129,7 @@ class CAR:
       b'\xf1\x878V0906264F \xf1\x890003',
       b'\xf1\x878V0906264L \xf1\x890002',
       b'\xf1\x878V0906264M \xf1\x890001',
-      b'\xf1\x878V09C0BB01 \xf1\x890001',  # Malformed ECU SW part number due to COBB/EQT tune
+      b'\xf1\x878V09C0BB01 \xf1\x890001',  # Corrupted by COBB/EQT tune
```

Comment:
remove the comment and just make a list of expected modifications in the comment

Reply:
Disagree:

1) I don't know that I can enumerate all expected modifications
2) If I have knowledge a particular FW entry is unreliable for vehicle ID and/or compatibility checks, I want to retain that

Frankly it pisses me off that I have to have that ECU entry at all. It's stupid and it's been several years since I've seen a reputable tuner do so. Having checked some other available scan data from Ross-Tech's forum, it looks like even COBB quit doing it on more recent tunes. I'm inquiring with the user to see if he can have COBB fix it so I don't have to.